go/stellar/stellar.go

```diff
@@ -515,6 +515,10 @@ func SendPaymentGUI(m libkb.MetaContext, walletState *WalletState, sendArg SendP
 func sendPayment(m libkb.MetaContext, walletState *WalletState, sendArg SendPaymentArg, isCLI bool) (res SendPaymentResult, err error) {
 	defer m.CTraceTimed("Stellar.SendPayment", func() error { return err })()
 
+	if err = m.Ctx().Err(); err != nil {
+		return res, err
+	}
```

Comment:
what's this block for?

Reply:
It's just a friendly yield. When context's get canceled sometimes it's nice not to have them log a huge tree before realizing they've been canceled at the bottom of the callstack. But I don't remember why this one in particular and it doesn't seem likely for this to be canceled, so I'll remove it.